### DISCUSSION TITLE
New feature marker properties, "engineering notation"

### DISCUSSION THREAD

**Discussion by cmat21** [ra3xdh/qucs_s#381] [Ideas]
Hi, could you consider adding the ability to display the "frequency" and "C value" using engineering notation?
![marker_properties](https://github.com/ra3xdh/qucs_s/assets/70080894/c2a85a3d-f7e9-4d3a-942a-5a38a5ecbe67)

**Comment by ra3xdh** [comment]
Yes, the marker and the Y-axis should use the same notation. I will add a fix soon.

**Comment by ra3xdh** [comment]
The fix will be available since the next release #383
